zebra/fpm_listener.c

```diff
@@ -432,7 +432,7 @@ static int parse_multipath_attr(struct netlink_msg_ctx *ctx,
 		uint32_t vxlan_vni;
 		uint16_t encap_type;
 
-		if (!RTNH_OK(rtnh, len)) {
+		if (!RTNH_OK(rtnh, (long)len)) {
```

Comment:
So do the RTNH_* macros expect lengths to be longs then? Is it more correct to change len to a long rather than cast it from int?